UefiPayloadPkg/UefiPayloadPkgIa32X64.dsc

```diff
@@ -397,6 +398,8 @@
   gEfiSecurityPkgTokenSpaceGuid.PcdTpmInstanceGuid|{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00}
   gEfiSecurityPkgTokenSpaceGuid.PcdTpmInitializationPolicy|0
 
+  gUefiPayloadPkgTokenSpaceGuid.PcdFchSpiBar|0xFEC10000

```

Comment:
@miczyg1 I'm not sure if this is good idea, something say to me this is terrible idea. How those kind of resources are discovered by MinPlatform code?

Reply:
@pietrushnic this is either way patched in the AmdSPi module to the value programmed in FCH. The DSC provides only sane default.